### ISSUE TITLE
PowerPointPresentation: Slide name not defined

### ISSUE THREAD

**Issue by Skaty** [PowerPointLabs/PowerPointLabs#1198]
When a new PowerPointPresentation object is created from an existing opened presentation, the name of the presentation is not defined in the constructor.

As such, any functions/properties that rely on the name, for instance Opened, will not work.

**Comment by kai33** [comment]
instead of fixing the name and comparing the name to verify it's open,
is there better way to verify that a ppt is open?

**Comment by Skaty** [comment]
@kai33

I don't think so. I guess the reason why the comparison is done is to ensure that the Model does not refer to a Presentation that is closed, as the Model is our abstraction of the VSTO Presentation.

**Comment by kai33** [comment]
for example, can there be more than 1 ppt document with the same name, and then mess up the logic?

**Comment by Skaty** [comment]
@kai33

Yes, that might occur. We need a better way to identify the presentations.

**Comment by kai33** [comment]
as discussed, `Opened` should use VSTO presentation's full name to verify

**Comment by Skaty** [comment]
@kai33 Use name or just catch exception (or both)?

**Comment by kai33** [comment]
Use full name and catch exception in case its resource is released
